nixos/modules/services/printing/lprint.nix

```
@@ -0,0 +1,520 @@
+          description = ''
+            The maximum number of completed jobs.
+          '';
+        };
+        defaultMediaCollection = lib.mkOption {
+          type = mediaCollectionType;
+          description = ''
+            The default media collection.
+          '';
+        };
+        additionalMediaCollections = lib.mkOption {
+          type = types.list mediaCollectionType;
+          default = [];
+          description = ''
+            Additional media collections.
+          '';
+        };
+        defaultOrientation = lib.mkOption {
+          type = types.nullOr (types.oneOf [ types.enum [
+            "portrait"
+            "landscape"
+            "reverse-landscape"
+            "reverse-portrait"
+            "none"
+          ] types.str ]);
+          default = "none";
+          description = ''
+            The default orientation.
+          '';
+        };
+        defaultColorMode = lib.mkOption {
+          type = types.oneOf [ types.enum [
+            "auto"
+            "color"
+            "monochrome"
+          ] types.str ];
+          default = "auto";
+          description = ''
+            The default color mode.
+          '';
+        };
+        defaultContentOptimize = lib.mkOption {
+          type = types.oneOf [ types.enum [
+            "auto"
+            "graphic"
+            "photo"
+            "text"
+          ] types.str ];
+          default = "auto";
+          description = ''
+            The default content optimize.
+          '';
+        };
+        defaultPrintQuality = lib.mkOption {
+          type = types.oneOf [ types.enum [
+            "draft"
+            "normal"
+            "high"
+          ] types.str ];
+          default = "normal";
+          description = ''
+            The default print quality.
+          '';
+        };
+        defaultPrintScaling = lib.mkOption {
+          type = types.oneOf [ types.enum [
+            "auto"
+            "fit"
+            "fill"
+            "none"
+          ] types.str ];
+          default = "auto";
+          description = ''
+            The default print scaling.
+          '';
+        };
+        defaultPrintSpeed = lib.mkOption {
+          type = types.int;
+          default = 0;
+          description = ''
+            The default print speed.
+          '';
+        };
+        printerDarkness = lib.mkOption {
+          type = types.int;
+          default = 50;
+          description = ''
+            The printer darkness.
+          '';
+        };
+        printerResolution = lib.mkOption {
+          type = types.str;
+          description = ''
+            The printer resolution.
+          '';
+        };
+        doubleSided = lib.mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Whether the printer supports double-sided printing.
+          '';
+        };
+      };
+    })
+  ];
+  lprintConfig = config.services.printing.lprint;
```

```suggestion
  cfg = config.services.printing.lprint;
```
this is more standard